kernel/queue.c

```diff
@@ -88,6 +88,9 @@ static inline void handle_poll_events(struct k_queue *queue, uint32_t state)
 {
 #ifdef CONFIG_POLL
 	z_handle_obj_poll_events(&queue->poll_events, state);
+#else
+	ARG_UNUSED(queue);
+	ARG_UNUSED(state);
```

Comment:
Don't put these inside #if conditions.  They're harmless in all contexts, just put them up at the top of the function where they don't get in the way.

Reply:
mmm fwiw I just looked at how it seemed to be done in other places in the kernel and e.g. `thread.c` definitely uses the pattern where `ARG_UNUSED` is inside #if conditions, so it should probably be consistent at least throughout a file?